charts/mirrorbits-parent/templates/serviceaccount.yaml

```
@@ -39,28 +37,26 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
-  name: {{ template "jenkins.serviceAccountName" . }}-rolebinding
+  name: {{ template "mirrorbits-parent.serviceAccountName" . }}-rolebinding
   namespace: {{ .Release.Namespace }}
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: Role
-  name: {{ template "jenkins.serviceAccountName" . }}-role
+  name: {{ template "mirrorbits-parent.serviceAccountName" . }}-role
 subjects:
 - kind: ServiceAccount
-  name: {{ template "jenkins.serviceAccountName" . }}
-  namespace: {{ template "jenkins.serviceAccountNamespace" . }}
+  name: {{ template "mirrorbits-parent.serviceAccountName" . }}
+  namespace: {{ template "mirrorbits-parent.serviceAccountNamespace" . }}
```

```suggestion
  namespace: {{- .Release.Namespace -}}
```

(Remnant of the "existingSvcAccount" template)